.github/workflows/server.yml

```diff
@@ -31,8 +31,6 @@ jobs:
           yarn install
 
       - name: Build
-        env:
-          JWT_PRIVATE_KEY: none
         run: |
           cd server
```

Comment:
Isn't it possible to set the working directory somewhere for this job?

Reply:
yes! here you go